src/lib/components/PrivateRoute/private-route.component.js

```diff
@@ -2,22 +2,24 @@ import React, { Component } from "react";
 import { withWebId } from "@solid/react";
 import { Redirect, Route } from "react-router-dom";
 
+
 import { Loader } from "./private-route.style";
 
+
 type Props = {
   webId?: String,
   redirect: String,
   component: Node,
   loaderComponent: Node
 };
 
-class PrivateRoute extends Component<Props> {
+export class PrivateRoute extends Component<Props> {
   renderRouter = (): React.Element => {
     const { webId, redirect, component: Component, ...rest } = this.props;
     return webId ? (
       <Route {...rest} component={Component} />
     ) : (
-      <Redirect to={redirect} />
+      <Redirect to={redirect} data-testid="redirect-component" />
```

Comment:
What is this doing?

Reply:
This is a way from react-test-library to get a component from DOM.